source/hardware/controllers/numark_dj2go2_touch.rst

```diff
@@ -0,0 +1,26 @@
+.. _numark-dj2go2-touch:
+
+Numark DJ2GO2 Touch
+===================
+
+.. sectionauthor::
+   tandy1000 <https://github.com/tandy-1000>
+
+The Numark DJ2GO2 Touch is a USB :term:`MIDI` controller with an integrated audio interface.
+It is the successor to the DJ2GO2, and features touch capable wheels.
+
+-  `Manufacturer’s product page <https://www.numark.com/product/dj2go2-touch>`__
+-  `Serato DJ Hardware Page <https://serato.com/dj/hardware/numark-dj2go2-touch>`__
+-  `Mapping Forum Thread <https://mixxx.discourse.group/t/a-complete-numark-dj2go2-touch-mapping/22616>`__
+
+.. versionadded:: 2.3.0
+
+Controller Mapping
+------------------
+
+Everything in this mapping is standard except from the Level knobs, which have been remapped to a filter effect.
```

Comment:
This is not correct, you implemented the custom preview seek functionality on the browse knob.

Reply:
Oops, just added that in